E2E-tests/config/substrate/ci_stack.json

```diff
@@ -4,7 +4,7 @@
         "ogmios_port": 1337,
         "kupo_host": "devnet-services-service",
         "kupo_port": 1442,
-        "tools_host": "10.0.12.108",
+        "tools_host": "10.0.12.126",
```

Comment:
if these change and need to be updated perhaps there's a better way? if that IP isn't static, maybe Skylar could help make it so